tests/sqllogictests/suites/mode/standalone/explain/09_0039_target_build_merge_into_standalone.test

```diff
@@ -316,5 +316,76 @@ select * from target_build_optimization order by a,b,c;
 11 b11 c_11
 12 b12 c_12
 
+## add explain analyze
+statement ok
+create table t1(a int);
+
+statement ok
+insert into t1 values(1),(1);
+
+statement ok
+insert into t1 values(2);
+
+statement ok
+create table t2(a int);
+
+statement ok
+insert into t2 values(2);
+
+query T
+select count(*) from fuse_block('default','t1');
+----
+1
+
+query T
+select count(*) from fuse_block('default','t2');
+----
+1
+
+query T
+explain analyze merge into t1 using t2 on t1.a = t2.a when matched then update * when not matched then insert *;
+----
+HashJoin
+├── output columns: [t1.a (#1), t1._row_id (#2), t2.a (#0)]
+├── join type: RIGHT OUTER
+├── build keys: [t2.a (#0)]
+├── probe keys: [t1.a (#1)]
+├── filters: []
+├── estimated rows: 1.50
+├── cpu time: 771µs
+├── wait time: 13.581625ms
```

Comment:
These infos may be different between different machines, such as local and ci env